README.md

```diff
@@ -38,6 +38,8 @@ Full documentation for releases can be found on [https://distro.eks.amazonaws.co
 To receive notifications about new EKS-D releases, subscribe to the EKS-D updates SNS topic:
 `arn:aws:sns:us-east-1:379412251201:eks-distro-updates`
 
+[<img src="docs/contents/certified-kubernetes-color.svg" height=150>](https://github.com/cncf/k8s-conformance/pull/2252)
+
 ### Kubernetes 1-24
```

Comment:
We'd technically be certified for 1.24 and 1.23 if the conformance results have been submitted, but the older releases have already reached EOL has far as the community is concerned. This unversion badge may cause confusion. We should have a badge here though